.github/workflows/publish_connectors.yml

```diff
@@ -264,6 +296,39 @@ jobs:
           SPEC_CACHE_GCS_CREDENTIALS: ${{ secrets.METADATA_SERVICE_DEV_GCS_CREDENTIALS }}
           METADATA_SERVICE_GCS_CREDENTIALS: ${{ secrets.METADATA_SERVICE_DEV_GCS_CREDENTIALS }}
 
+      - name: Generate OSS Registry Entry [Manual]
+        id: generate-oss-registry-entry-manual
+        if: github.event_name == 'workflow_dispatch' || github.event_name == 'workflow_call'
+        shell: bash
+        run: |
+          echo "Generating OSS registry entry for ${{ matrix.connector }}"
+          poetry run --directory airbyte-ci/connectors/metadata_service/lib \
+            metadata_service generate-registry-entry \
+            prod-airbyte-cloud-connector-metadata-service \
+            "airbyte-integrations/connectors/${{ matrix.connector }}/metadata.yaml" \
+            "spec.json" \
+            oss
```

Comment:
I feel like this needs something to trigger prerelease mode?

Reply:
fixed in https://github.com/airbytehq/airbyte/pull/64914/commits/8fd39f7a6b3a6fa51608460aad18a5307d3cc2ca